Phaethon/Tests/IntegrationTests/LoginTest.cs

```diff
@@ -1,78 +1,78 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Collections.Immutable;
-using System.Data.Entity;
-using System.Data.Entity.Core.Objects;
 using System.Linq;
 using System.Net;
 using System.Net.Http;
-using System.Runtime.ExceptionServices;
 using System.Text;
 using System.Threading.Tasks;
 using Core.Model;
 using InternalApi.DataAccess;
-using Newtonsoft.Json;
 using NUnit.Framework;
 using System.Security.Cryptography;
-using InternalApi.DataManagement;
 
 namespace Tests.IntegrationTests
 {
-
    public class LoginTest : InternalTestFakeServerBase
     {
-        private Login userModel;
-        private Login userModel2;
-        private int count1;
-        private int count2;
-        private string name;
-        private string name2;
-        private string OGPass;
-        private LoginDa loginDa;
+        private Login _userModel;
```

Comment:
https://github.com/RaidenRabit/Phaethon/blob/99c8a620e6cbbbe3d1c4f49d3a255f08cea6d221/Phaethon/Tests/IntegrationTests/LoginTest.cs#L18-L21
These variables should be local variables, not class wide variables, because they are used only in 1-2 tests, each.